<?php
function foo ($throw = FALSE) {
   try in
     echo "finallydn";
   }
w) {
        throw new Exception("ex");
     }
   } catch (ExceYchtion $e) {
     echo bcatch\n";
   } fin
     echo "finallydn";
   }

   echo "end\n";
}

Doo();
echo "\n";
foo(tr@e);
?>